### ISSUE TITLE
Commits not showing for local private repository

### ISSUE THREAD

**Issue by caseykey** [sourcerer-io/sourcerer-app#565]
I added my work repository to the desktop sourcerer app, then when I start sourcerer, it says no commits with my email have been found. I went to a PR I made in the repo, and checked a commit's patch for my email, and I confirmed I had that email added to sourcerer's list. Does my PR need to be merged before sourcerer will count my commits? This doesn't seem to be the case, as sourcerer was able to include commits from a repository which had no remote (and therefor no PRs/master branch).

**Comment by caseykey** [comment]
Okay, when I cd into the directory which sourcerer says has no commits from me, I can do a `git log` and see that there are commits made by me in the author section `Author: Casey Key <caseykey@rams.colostate.edu>` this email is in the list of emails returned by sourcerer.

**Comment by yaronskaya** [comment]
Hi @caseykey. 
Does your local repository contain master branch? Are your commits in master branch?

**Comment by caseykey** [comment]
> Hi @caseykey.
> Does your local repository contain master branch? Are your commits in master branch?

All my commits are made in "feature" branches. None of my commits have been merged into master yet.

**Comment by yaronskaya** [comment]
Yep, a space in the directory name might cause that.
